packages/common-types/src/earn.ts

```diff
@@ -97,19 +187,50 @@ export type BaseDetailedStrategy = BaseStrategy & {
   detailed: true;
 };
 
+export type SavedBaseSdkStrategy = SdkBaseStrategy & { lastUpdatedAt: number };
+export type SavedBaseDetailedSdkStrategy = SdkBaseDetailedStrategy & { detailed: true; lastUpdatedAt: number };
+export type SavedSdkStrategy = SavedBaseSdkStrategy | SavedBaseDetailedSdkStrategy;
 export type Strategy = BaseStrategy | BaseDetailedStrategy;
 
-export type SummarizedSdkStrategyParameters = {
-  farms: Record<FarmId, StrategyFarm>;
-  guardians: Record<GuardianId, StrategyGuardian>;
-  tokens: {
-    assets: Record<TokenListId, SdkStrategyToken>;
-    rewards: Record<TokenListId, SdkStrategyToken>;
-  };
-  networks: Record<number, NetworkStruct>;
-  yieldTypes: StrategyYieldType[];
+export type SavedBaseSdkEarnPosition = BaseSdkEarnPosition & { lastUpdatedAt: number };
+export type SavedBaseDetailedSdkEarnPosition = DetailedEarnPosition & { detailed: true; lastUpdatedAt: number };
+export type BaseSavedSdkEarnPosition = SavedBaseSdkEarnPosition | SavedBaseDetailedSdkEarnPosition;
+export type SavedSdkEarnPosition = Omit<BaseSavedSdkEarnPosition, 'strategy'> & {
+  strategy: StrategyId;
 };
 
-export type GuardianId = string;
-export type FarmId = string;
-export type StrategyId = string;
+export type BaseEarnPosition = {
+  id: SdkEarnPositionId;
+  createdAt: Timestamp;
+  owner: Address;
+  permissions: EarnPermissions;
+  strategy: Strategy;
```

Comment:
I'm thinking if we could have one strategy object. Like we did for details, merging it into base-strategy, what if we also merge position-data into the allStrategies? We would avoid having duplicated data, and it would be easier to check if a strategy is a position or not, without calling the two lists (like in earn home, or in vault details). Not sure if you have already explored this option, let me know